model/fn-execution/src/main/proto/org/apache/beam/model/fn_execution/v1/beam_fn_api.proto

```diff
@@ -761,6 +761,13 @@ message Elements {
   // extensible and backward compatible
   message ElementMetadata {
     optional DrainMode.Enum drain = 1;
+    // (Optional) As part of https://www.w3.org/TR/trace-context/ we are forwarding a trace and participating in it.
+    // Traceparent header represents the incoming request in a tracing system in a common format.
+    // Example value: 00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01
+    optional string traceparent = 2;
+    // (Optional) tracestate extends traceparent with vendor-specific data represented by a set of name/value pairs.
```

Comment:
If it is always name/value pairs, should we use a map?
If it is just vendor specific maybe it should just omit that it is set of name/value pairs here or otherwise say for example opentelementry uses this format.